components/core/DownloadDaemon.py

```diff
@@ -213,6 +219,25 @@ def find_supported_handler(download):
             return handler
     return None
 
+
+def send_file_details(file_name, file_path):
+    m_fname = file_name
+    m_fpath = file_path
+    upload_to_minio(m_fname, m_fpath)
+    print("File uploaded to minio/bassa successfully !")
+
+
+def send_details_to_minio(file_name, file_path, gid, completedLength, username, download_id):
+    m_fname = file_name
+    m_fpath = file_path
+    m_gid = gid
+    m_size = completedLength
+    m_username = username
+    m_id = download_id
+    update_minio_indexes(m_fname, m_fpath, m_gid, m_size, m_username, m_id)
+    print("Database table of minio updated successfully !")
```

Comment:
Instead of print, log it. Also, remove the `m_**` variables and directly use function argument values